### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by notpaz** [VolmitSoftware/Gloss#73]
Plugins (51): MotdManager, Mortar, OreBroadcast, GroupManager, PlugMan, Announcer, MaxBans, PotionFix, ViaVersion, CratesPlus, PeachReport, LightningDeath, killStats, Essentials, AltsRevealer, Sloth, ClearLag, IllegalStack, WorldBorder, BuycraftX, Votifier, Freeze, ProtocolLib, GuiRedeemMCMMO, KitAdder, FastAsyncWorldEdit, ObsidianBreaker, EssentialsChat, SuperbVote, mcMMO, Vault, PlayerParticles, MassiveCore, ViaRewind, EnderPearlCoolDown, HideAndCustomPlugins, Spartan, PlayerVaults, Wormholes, EssentialsSpawn, WorldEdit, Gloss, WorldGuard, Factions, FactionsTop, GlossChatBubbles, ChatControl, GlossDamageIndicators, SilkSpawners, CombatTagPlus, FactionChat
```
> [06:58:30 ERROR]: stopTiming called async for world - chunkLoad - Entities
> [06:58:30 WARN]: java.lang.Throwable
> [06:58:30 WARN]: at co.aikar.timings.TimingHandler.stopTiming(TimingHandler.java:114)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkRegionLoader.loadEntities(ChunkRegionLoader.java:424)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:46)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:1)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:337)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:295)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:16)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:135)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:117)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:145)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:604)
> [06:58:30 WARN]: at org.bukkit.Location.getChunk(Location.java:78)
> [06:58:30 WARN]: at com.volmit.gloss.api.display.LitSource.setPosition(LitSource.java:51)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.setLocation(GlossDescriptor.java:104)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.fromJSON(GlossDescriptor.java:195)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.<init>(GlossDescriptor.java:48)
> [06:58:30 WARN]: at com.volmit.gloss.controllers.SourceController.load(SourceController.java:149)
> [06:58:30 WARN]: at com.volmit.gloss.controllers.SourceController$1.run(SourceController.java:191)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
> [06:58:30 WARN]: at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
> [06:58:30 WARN]: at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
> [06:58:30 WARN]: at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
> [06:58:30 WARN]: at java.lang.Thread.run(Thread.java:748)
> [06:58:30 ERROR]: stopTiming called async for world - chunkLoad - TileEntities
> [06:58:30 WARN]: java.lang.Throwable
> [06:58:30 WARN]: at co.aikar.timings.TimingHandler.stopTiming(TimingHandler.java:114)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkRegionLoader.loadEntities(ChunkRegionLoader.java:438)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:46)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:1)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:337)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:295)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:16)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:135)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:117)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:145)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:604)
> [06:58:30 WARN]: at org.bukkit.Location.getChunk(Location.java:78)
> [06:58:30 WARN]: at com.volmit.gloss.api.display.LitSource.setPosition(LitSource.java:51)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.setLocation(GlossDescriptor.java:104)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.fromJSON(GlossDescriptor.java:195)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.<init>(GlossDescriptor.java:48)
> [06:58:30 WARN]: at com.volmit.gloss.controllers.SourceController.load(SourceController.java:149)
> [06:58:30 WARN]: at com.volmit.gloss.controllers.SourceController$1.run(SourceController.java:191)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
> [06:58:30 WARN]: at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
> [06:58:30 WARN]: at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
> [06:58:30 WARN]: at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
> [06:58:30 WARN]: at java.lang.Thread.run(Thread.java:748)
> [06:58:30 ERROR]: stopTiming called async for world - chunkLoad - TileTicks
> [06:58:30 WARN]: java.lang.Throwable
> [06:58:30 WARN]: at co.aikar.timings.TimingHandler.stopTiming(TimingHandler.java:114)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkRegionLoader.loadEntities(ChunkRegionLoader.java:459)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:46)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOProvider.callStage2(ChunkIOProvider.java:1)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.skipQueue(AsynchronousExecutor.java:337)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.util.AsynchronousExecutor.getSkipQueue(AsynchronousExecutor.java:295)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.chunkio.ChunkIOExecutor.syncChunkLoad(ChunkIOExecutor.java:16)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:135)
> [06:58:30 WARN]: at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:117)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:145)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getChunkAt(CraftWorld.java:604)
> [06:58:30 WARN]: at org.bukkit.Location.getChunk(Location.java:78)
> [06:58:30 WARN]: at com.volmit.gloss.api.display.LitSource.setPosition(LitSource.java:51)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.setLocation(GlossDescriptor.java:104)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.fromJSON(GlossDescriptor.java:195)
> [06:58:30 WARN]: at com.volmit.gloss.context.GlossDescriptor.<init>(GlossDescriptor.java:48)
> [06:58:30 WARN]: at com.volmit.gloss.controllers.SourceController.load(SourceController.java:149)
> [06:58:30 WARN]: at com.volmit.gloss.controllers.SourceController$1.run(SourceController.java:191)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:59)
> [06:58:30 WARN]: at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
> [06:58:30 WARN]: at org.github.paperspigot.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:23)
> [06:58:30 WARN]: at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
> [06:58:30 WARN]: at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
> [06:58:30 WARN]: at java.lang.Thread.run(Thread.java:748)
```

**Comment by DavidRockin** [comment]
Does the plugin cease to function? I'm positive it might be related when a hologram attempts to load in an unloaded world.

**Comment by notpaz** [comment]
There's only the default worlds with holograms at spawn, and the plugin keeps working. If it helps, I imported from holographicdisplays.

**Comment by DavidRockin** [comment]
What are your gloss and mortar versions

**Comment by DavidRockin** [comment]
This has been fixed in 1.3.0, occurs when Gloss attempts to load a hologram in an unloaded world.
